SPEC.md

```diff
@@ -856,3 +862,6 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
     * **readOnly** (boolean, optional if **kind** is "host") whether or not the volume should be mounted read only.
 * **isolators** (list of objects, optional) list of isolators that will apply to all apps in this container. Each object has two key value pairs: **name** is restricted to the AC Name formatting and **value** can be a freeform string)
 * **annotations** (list of objects, optional) arbitrary metadata the executor should make available to applications via the metadata service. Objects must contain two key-value pairs: **name** is restricted to the [AC Name](#ac-name-type) formatting and **value** is an arbitrary string). Annotation names must be unique within the list.
+* **ports** (list of objects, optional) list of ports that will be exposed on the host.
+    * **name** (string, required) name the port in the image manifest that should be exposed on the host (restricted to the AC Name formatting).
+    * **hostPort** (integer, required) the port number on the host that will be mapped to the container port.
```

Comment:
@alban not at all. The "name" references a port defined in the image manifest which contains (protocol, container port) information. hostPort actually allows for the host port to be distinct from the container port.